src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Feedback/HTMLCSharpLanguageServerFeedbackFileLoggerProvider.cs

```diff
@@ -15,6 +15,11 @@ internal class HTMLCSharpLanguageServerFeedbackFileLoggerProvider : ILoggerProvi
 
         private readonly FeedbackFileLoggerProvider _loggerProvider;
 
+        // Internal for testing
+        public HTMLCSharpLanguageServerFeedbackFileLoggerProvider()
```

Comment:
I assume you meant to make this constructor internal?

Reply:
Haha, yes, thanks!

```suggestion
        internal HTMLCSharpLanguageServerFeedbackFileLoggerProvider()
```